### DISCUSSION TITLE
Revert changes

### DISCUSSION THREAD

**Discussion by LeontGreek** [ionuttbara/windows-defender-remover#135] [Q&A]
Hello!

I have applied the remover, but I didn't make a restore point. Is it possible to revert somehow the changes and reinstall all these components? As the last way I think to reinstall windows 11, but I don't want it, because I have many installed programs and settings.

Thank you in advance!

**Comment by xavierOu812** [comment]
I am on Windows 10 21H2. Last night I set a restore point to test Defender Remover version 12.7.2. After using the OS for a bit I used system restore to bring back Defender. The restore did not work and Defender was still removed. Whatever 12.7.2 does System Restore cannot revert.
I was able to bring back Defender today by using Defender Remover 12.5. I noticed in the release notes that there are options to restore Defender in this version.
Run 12.5 as administrator and choose option R. Other versions may have this menu as well. If 12.7.2. has it I don't know how to access it.
![Defender 12 5](https://github.com/ionuttbara/windows-defender-remover/assets/165600302/f6157594-9ed6-45bf-82ab-d2477b14d666)

**Reply by annarana** [reply]
I tried to do that on windows 11 but doesn't work. Do you have any other suggestions?

**Reply by xavierOu812** [reply]
Back up data.
Reinstall operating system.

**Comment by darknessdoctor** [comment]
i can't revert too same my restore point got messed up and others apps now are infected too i can't use windows i need to revert this any other script to enable it i have the 'threat and protection tab misssing

**Comment by lowercaselowen** [comment]
thanks to all this, im just going to image the system first - then apply this fix - I have several crypto systems that are stand alone and need minimal defense software

**Comment by PanBambaryla** [comment]
I was able to recover with 3 x **Windows 11 PRO 23H2** machines in the following way:
1) get a working PC with the same system version and export **HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Services\wscsvc**
2) import this non-existent key on the machine which has defender removed
3) do the same with the key: **HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Services\SecurityHealthService**
4) run as admin **sfc /scannow** which do some repairs
5) run as admin **DISM.exe /Online /Cleanup-image /Restorehealth**
6) reboot your PC
7) do a quick Windows Update to get a small Windows Security Update which does not require reboot

Not sure if it was relevant but I think I have repaired and reset Windows Security after step 3 like in the following link:

https://www.thewindowsclub.com/reset-windows-security-app-in-windows-10

and voila - security center is opening again, no more problems.

**Comment by dohn** [comment]
I would like an answer from the author of this program.

**Comment by ghost** [comment]
I've done just about everything mentioned here, my Defender page is hidden and I cannot turn off Defender anymore if I access it through other means. 
![ApplicationFrameHost_Rvl8UNM02k](https://github.com/user-attachments/assets/01966db9-e0ad-4f45-9ce8-682bdde6d1ac)


# It should be mentioned in **BIG BOLD LETTERS** that one cannot EASILY restore Defender afterwards. 
A system restore point is not going to save anyone. Reinstalling the OS is the only option to avoid intense headaches from trying to revert the changes made by the REG file.

I do wonder why older versions had more options (turn off, turn on) instead of just having an option to completely remove Defender.

**Reply by ghost** [reply]
In any case, thanks to everyone who mentioned their solutions here. Windows Security does work and registers my Malwarebytes software fine.

**Comment by gramwilliam123** [comment]
It’s like hitting a “reset” on what you just changed.
